public/themes/custom/hdbt_subtheme/translations/sv.po

```diff
@@ -3,3 +3,9 @@ msgstr "Enhetens hemsida"
 
 msgid "Additional details"
 msgstr "Ytterligare detaljer"
+
+msgctxt "Health station search count"
+msgid "health station"
+msgid_plural "health stations"
+msgstr[0] "hälsostation"
+msgstr[1] "hälsostation"
```

Comment:
Is this really grammatically correct or should it be for example _-ner_ ending? 🤔

Reply:
Changed this to hälsostationer